src/content/docs/accounts/accounts-billing/account-structure/multitenant-organizations.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Multi-tenant New Relic organizations
+tags:
+  - Accounts
+  - Account setup
+metaDescription: Options available for New Relic multi-tenant organizations, used by large enterprise customers and MSPs.  
+redirects:
+---
+
+Our multi-tenancy feature lets you have a New Relic organization that contains multiple sub-organizations within it. This is useful for large organizations that have departments they want to keep separate, and for managed service providers (MSPs) who need multiple tenant boundaries for their end users. 
```

Comment:
Suggested: Multi-tenancy allows New Relic customers to benefit from data segregation through setting up multiple organizations. This is useful for large enterprises that have business units they want to keep separate, or for managed service providers (MSPs) who need to establish tenant boundaries between their end customers. 

---
Trying to avoid language around sub-organizations since it implies a hierarchy. Even though there is a concept of managing org and member orgs, there's no inheritance concept in V2 user model.